app/views/includes/layout.njk

```diff
@@ -33,7 +36,7 @@
     {% block outerBody %}{% endblock %}
 
     <div class="nhsuk-width-container">
-      <main class="nhsuk-main-wrapper" id="maincontent">
+      <main class="nhsuk-main-wrapper app-main-wrapper" id="maincontent">
```

Comment:
There are no styles for the class `app-main-wrapper`, this looks like a class used specifically in the NHS digital service manual app